test/index.js

```diff
@@ -7,9 +7,9 @@ import { normalize, Schema, arrayOf, unionOf } from 'normalizr';
 
 describe("denormalize", () => {
 
-  it("should return undefined when denormalizing an undefined entity", () => {
-    expect(denormalize(undefined)).to.be.undefined;
-  });
+  // it("should return undefined when denormalizing an undefined entity", () => {
+  //   expect(denormalize(undefined)).to.be.undefined;
+  // });
```

Comment:
Whoops, I had commented out some passing specs to more easily see console logs in failing ones. Looks like I had forgotten to uncomment this one. Fixed!